### ISSUE TITLE
How would I initialize a RepeatingGroup?

### ISSUE THREAD

**Issue by Moataz-E** [quickfixgo/quickfix#328]
So, my FIX specification generated the following code:

> func NewNoMDEntriesRepeatingGroup() NoMDEntriesRepeatingGroup {
	return NoMDEntriesRepeatingGroup{
		quickfix.NewRepeatingGroup(tag.NoMDEntries,
			quickfix.GroupTemplate{quickfix.GroupElement(tag.MDUpdateAction), quickfix.GroupElement(tag.MDEntryType), quickfix.GroupElement(tag.MDStreamID), quickfix.GroupElement(tag.SecurityID), quickfix.GroupElement(tag.SecurityIDSource), quickfix.GroupElement(tag.Product), quickfix.GroupElement(tag.SecurityDesc), NewNoInstrumentPartiesRepeatingGroup(), NewNoUnderlyingsRepeatingGroup(), NewNoLegsRepeatingGroup(), quickfix.GroupElement(tag.MDEntryPx), quickfix.GroupElement(tag.PriceType), quickfix.GroupElement(tag.YieldType), quickfix.GroupElement(tag.Yield), quickfix.GroupElement(tag.Spread), quickfix.GroupElement(tag.SpreadType), quickfix.GroupElement(tag.BenchmarkPrice), quickfix.GroupElement(tag.BenchmarkPriceType), quickfix.GroupElement(tag.BenchmarkSecurityID), quickfix.GroupElement(tag.BenchmarkSecurityIDSource), quickfix.GroupElement(tag.BenchmarkSecurityDesc), quickfix.GroupElement(tag.UpfrontPrice), quickfix.GroupElement(tag.UpfrontPriceType), quickfix.GroupElement(tag.MDEntrySize), quickfix.GroupElement(tag.MDEntryDate), quickfix.GroupElement(tag.OpenInterest), quickfix.GroupElement(tag.ExpireTime), quickfix.GroupElement(tag.QuoteEntryID), quickfix.GroupElement(tag.PriceDelta), quickfix.GroupElement(tag.NetChgPrevDay), quickfix.GroupElement(tag.TradeVolume), quickfix.GroupElement(tag.TransactTime), quickfix.GroupElement(tag.AggressorSide), quickfix.GroupElement(tag.MDQuoteType), quickfix.GroupElement(tag.PriceScalingFactor), quickfix.GroupElement(tag.VolumeScalingFactor)})}
}

How would I create a new object and assign values to the `MDEntryType`, `SecurityID`, `Product`, etc...

In other words, once I've initialized a `NoMDEntriesRepeatingGroup` object using the `NewNoMDEntriesRepeatingGroup` function, how do I modify that object?

**Comment by cbusbey** [comment]
The generated code will include an `Add` function on NoMDEntriesRepeatingGroup.  Call this to create a new group within the repeating group and call the Set functions on this group to initialize values.

**Comment by Moataz-E** [comment]
Perfect, thank you!
